src/build/packages.rs

```diff
@@ -219,19 +220,25 @@ fn read_dependencies<'a>(
         // Read all bsconfig files in parallel instead of blocking
         .par_iter()
         .map(|package_name| {
-            let path =
-                match PathBuf::from(helpers::get_relative_package_path(package_name, false)).canonicalize() {
-                    Ok(dir) => dir.to_string_lossy().to_string(),
-                    Err(e) => {
-                        print!(
-                          "{} {} Error building package tree (are node_modules up-to-date?)... \n More details: {}",
-                          style("[1/2]").bold().dim(),
-                          CROSS,
-                          e.to_string()
-                        );
-                        std::process::exit(2)
+            let path_buf =
+                match PathBuf::from(format!("{}/node_modules/{}", parent_path, package_name)).canonicalize() {
+                    Ok(p) => p,
+                    Err(_) => {
+                      match PathBuf::from(format!("node_modules/{}", package_name)).canonicalize() {
```

Comment:
we should use the absolute path and prepend with `project_root` here.

Reply:
Done